localization/pt/abstract-factory/README.md

```diff
@@ -0,0 +1,233 @@
+---
+título: Abstract Factory
+categoria: Criacional
+idioma: pt
+tags:
+- Gang of Four
```

Comment:
I'm sorry but you are not supposed to translate this. You should define it exactly like in the original pattern:

```
---
title: Abstract Factory
category: Creational
language: en
tags:
 - Gang of Four
---
```

Reply:
Except for the language of course which should be `pt` like you already have